pkg/common/utils/utils.go

```diff
@@ -223,3 +226,61 @@ func QueryVolumeDetailsUtil(ctx context.Context, m cnsvolume.Manager, volumeIds
 	}
 	return volumeDetailsMap, nil
 }
+
+// LogoutAllSessions will loggout all vCenter sessions and disconnect vCenter client
+func LogoutAllSessions(ctx context.Context, isMultiVCenterFssEnabled bool) error {
```

Comment:
Can we skip fetching and supplying fss while we are exiting container?

Reply:
made changes as suggested in latest commit